components/scream/src/physics/cld_fraction/cld_fraction_main_impl.hpp

```diff
@@ -29,13 +29,13 @@ ::main(
     const Int i = team.league_rank();
 
     const auto oqi   = ekat::subview(qi,   i);
-    const auto oalst = ekat::subview(alst, i);
-    const auto oaist = ekat::subview(aist, i);
-    const auto oast  = ekat::subview(ast,  i);
+    const auto oliq_cld_frac = ekat::subview(liq_cld_frac, i);
```

Comment:
Definitely not for this PR, but I think at some point we need to change the names of the single-col vars. What is the `o` in front supposed to mean?

Reply:
??? I don't know.  I took this syntax from the p3_main_impl code.

Reply:
I think the o was just someone's lazy attempt to have multiple versions of the same variable name (to include both scalar and vector versions). I've never figured out what the o means. Maybe output? I agree this should be cleaned up... in another PR.

Reply:
Conrad had a nice approach in the shoc code to use `_s` trailing after the variable name, presumably meaning "subview".